helpers.ts

```diff
@@ -52,6 +52,17 @@ export function trackDownloadProgress(destinationStream: NodeJS.WritableStream,
 	return progressStream;
 }
 
+export function isBuildFromCLI(item: string) {
```

Comment:
this function needs a return type.
Also its name is misleading. The implementation checks if the passed string contains the string `.aar` or `include.gradle`.